backend/code_review_backend/issues/api.py

```diff
@@ -48,7 +52,25 @@ def perform_create(self, serializer):
         serializer.save(diff=Diff.objects.get(pk=self.kwargs["diff_id"]))
 
 
+class IssueCheckStats(generics.ListAPIView):
+    """
+    List all analyzer checks per repository aggregated with
+    their total number of issues
+    """
+
+    serializer_class = IssueCheckSerializer
+    queryset = (
+        Issue.objects.values("analyzer", "check", "diff__revision__repository__slug")
```

Comment:
What do we do for Issues not linked to diffs?

Reply:
repository falls back to None in the output ([see added unit test](https://github.com/mozilla/code-review/pull/190/commits/e41ad9bedb5ae9b1edca3be1b28737b1cc721d7d#diff-e22f97f7cbc51441508e5c7e52414aecR198))

Reply:
Maybe we should have a direct link between issue and repository, as issues should not need a diff to be linked to a repository